microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -284,9 +286,11 @@ async function renderWidget(widget) {
 }
 ```
 
+
+<!-- ------------------------------ -->
 #### Update widgets on service worker updates
 
-When the service worker code changes in a PWA, the browser detects that change, installs the new service worker, and then later activates it.
+When the service worker code changes in a PWA, the browser detects that change, installs the new service worker,<!-- todo: review: is install & service worker sufficiently decoupled? --> and then later activates it.
```

Comment:
removing todo (ok?), & changing "it" to "the service worker"
@captainbrosset

Reply:
```
When the service worker code changes in a PWA, 
the browser detects that change, installs the new service worker, 
and then later activates the service worker.
```

Reply:
Yes, please make that change and remove the todo.

Reply:
fresh Files Changed tab confirms this was done